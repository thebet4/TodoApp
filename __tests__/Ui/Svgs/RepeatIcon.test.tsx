import renderer from 'react-test-renderer';

const RepeatIcon = require('../../../src/Ui/Sgvs/RepeatIcon').default

describe('<RepeatIcon />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(RepeatIcon).toJSON()
    expect(tree).toMatchSnapshot()
  });
});
