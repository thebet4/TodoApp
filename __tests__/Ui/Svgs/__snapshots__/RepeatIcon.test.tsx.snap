// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RepeatIcon /> renders correctly 1`] = `null`;
